<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtsqueeze3.qrc">
    <normaloff>:/img/lib/images/squeezeplayer.png</normaloff>:/img/lib/images/squeezeplayer.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="controlFrame">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>1</y>
      <width>771</width>
      <height>306</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>771</width>
      <height>306</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblSlimDisplay">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>4</y>
       <width>751</width>
       <height>84</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>751</width>
       <height>81</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QFrame" name="keypadFrame">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>90</y>
       <width>161</width>
       <height>209</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QPushButton" name="btn9">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>104</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_9.png</normalon>
        <selectedon>:/img/lib/images/btn_9_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn2">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>6</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_2.png</normalon>
        <selectedon>:/img/lib/images/btn_2_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn4">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>55</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_4.png</normalon>
        <selectedon>:/img/lib/images/btn_4_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn3">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>6</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_3.png</normalon>
        <selectedon>:/img/lib/images/btn_3_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn6">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>55</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_6.png</normalon>
        <selectedon>:/img/lib/images/btn_6_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn8">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>104</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_8.png</normalon>
        <selectedon>:/img/lib/images/btn_8_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn5">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>55</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_5.png</normalon>
        <selectedon>:/img/lib/images/btn_5_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn1">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>6</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_1.png</normalon>
        <selectedon>:/img/lib/images/btn_1_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn7">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>104</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_7.png</normalon>
        <selectedon>:/img/lib/images/btn_7_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn0">
      <property name="geometry">
       <rect>
        <x>54</x>
        <y>154</y>
        <width>51</width>
        <height>50</height>
       </rect>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/btn_0.png</normalon>
        <selectedon>:/img/lib/images/btn_0_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="arrowKeyFrame">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>90</y>
       <width>244</width>
       <height>210</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QPushButton" name="btnNowPlaying">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>149</y>
        <width>234</width>
        <height>51</height>
       </rect>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/nowPlaying.png</normalon>
        <selectedon>:/img/lib/images/nowPlaying_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLeftArrow">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>51</y>
        <width>117</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/leftArrow.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpArrow">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>2</y>
        <width>234</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/upArrow.png</normalon>
        <selectedon>:/img/lib/images/upArrow_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDownArrow">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>96</y>
        <width>234</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/downArrow.png</normalon>
        <selectedon>:/img/lib/images/downArrow_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRightArrow">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>51</y>
        <width>117</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/rightArrow.png</normalon>
        <selectedon>:/img/lib/images/rightArrow_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="controlFrame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>331</width>
       <height>210</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QPushButton" name="btnPower">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>55</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Power</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStop">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/stop.png</normalon>
        <selectedon>:/img/lib/images/stop_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpVolume">
      <property name="geometry">
       <rect>
        <x>204</x>
        <y>60</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/volup.png</normalon>
        <selectedon>:/img/lib/images/volup_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMute">
      <property name="geometry">
       <rect>
        <x>274</x>
        <y>60</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/mute.png</normalon>
        <selectedon>:/img/lib/images/mute_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRepeat">
      <property name="geometry">
       <rect>
        <x>57</x>
        <y>120</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>Repeat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDownVolume">
      <property name="geometry">
       <rect>
        <x>154</x>
        <y>60</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/voldown.png</normalon>
        <selectedon>:/img/lib/images/voldown_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdd">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/add.png</normalon>
        <selectedon>:/img/lib/images/add_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRewind">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/rewind.png</normalon>
        <selectedon>:/img/lib/images/rewind_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFForward">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/fforward.png</normalon>
        <selectedon>:/img/lib/images/fforward_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSize">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>120</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnShuffle">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>120</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>Shuffle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPlay">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/play.png</normalon>
        <selectedon>:/img/lib/images/play_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPause">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/lib/images/pause.png</normalon>
        <selectedon>:/img/lib/images/pause_selected.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSleep">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>120</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>Sleep</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBright">
      <property name="geometry">
       <rect>
        <x>111</x>
        <y>120</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>Bright</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetup">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>54</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Setup</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="cfPlaylistWidget" native="true">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>316</y>
      <width>154</width>
      <height>138</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="cfArtistSelectWidget" native="true">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>320</y>
      <width>154</width>
      <height>138</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="cfAlbumSelectWidget" native="true">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>328</y>
      <width>154</width>
      <height>138</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnRewind</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnPlay</tabstop>
  <tabstop>btnPause</tabstop>
  <tabstop>btnFForward</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnDownVolume</tabstop>
  <tabstop>btnUpVolume</tabstop>
  <tabstop>btnMute</tabstop>
  <tabstop>btnUpArrow</tabstop>
  <tabstop>btnRightArrow</tabstop>
  <tabstop>btnDownArrow</tabstop>
  <tabstop>btnLeftArrow</tabstop>
  <tabstop>btnNowPlaying</tabstop>
  <tabstop>btn0</tabstop>
  <tabstop>btn1</tabstop>
  <tabstop>btn2</tabstop>
  <tabstop>btn3</tabstop>
  <tabstop>btn4</tabstop>
  <tabstop>btn5</tabstop>
  <tabstop>btn6</tabstop>
  <tabstop>btn7</tabstop>
  <tabstop>btn8</tabstop>
  <tabstop>btn9</tabstop>
  <tabstop>btnShuffle</tabstop>
  <tabstop>btnRepeat</tabstop>
  <tabstop>btnBright</tabstop>
  <tabstop>btnSize</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>btnSleep</tabstop>
  <tabstop>btnPower</tabstop>
 </tabstops>
 <resources>
  <include location="qtsqueeze3.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>false</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
